Notifier-service/src/services/streams/conferenceStream.ts

```diff
@@ -41,14 +44,83 @@ export class ConferenceStreamMongo extends ConferenceStream {
             .pipe(
                 share()
             )
+        this.insertstreamObs$ = this.getStream().pipe(filter(data => {
```

Comment:
* Can you check from the RxJs doc, whether this we will require a share operator after the filter. Since getStream already provides a shared subject. So just check whether we need to provide share again or not.

Reply:
In the docs i couldn't find anywhere mentioned to use share after filtering also as share() is being used for multi casting so it won't require to mention it again.